app/src/common/shared/com/igalia/wolvic/ui/widgets/KeyboardWidget.java

```diff
@@ -877,7 +877,7 @@ private void handleDomainChange(KeyboardSelectorView.Item aItem) {
         updateCandidates();
     }
 
-    private void handleLanguageChange(KeyboardInterface aKeyboard) {
+    private void handleLanguageChange(KeyboardInterface aKeyboard, boolean remember) {
```

Comment:
Adding booleans to a method signature is not generally the best idea because that forces developers to always check the method implementation/docs in order to know what a given `callMethod(true,true,false)` does.

I'd suggest defining and enumerated type with values like Remember::Yes and Remember::No which are much more explicit in the call sites.